windows/benji.py

```diff
@@ -417,6 +418,67 @@ def main():
 			speak.say("Sorry Graph can not be Plotted")
 			speak.runAndWait()
 
+	#Box Office Status
+	elif link[-1] == "boxoffice":
+		try:
+			url = "https://in.bookmyshow.com/" + link[0] + "/movies/nowshowing"
+			r =  requests.get(url)
+			soup = BeautifulSoup(r.content, 'html.parser')
+
+			soup_level2 = []
+			show_status_list = []
+			shows_list = soup.find_all('div', attrs={'class': 'card-container wow fadeIn movie-card-container'})
+			for i in shows_list:
+				start = str(i).index("href=")
+				end = str(i).index("title=")
+				soup_level2.append("https://in.bookmyshow.com" + str(i)[start+6 : end-2])
+
+			show_status_raw = soup.find_all('div', attrs={'class': 'popularity sa-data-plugin'})
+			for i in show_status_raw:
+				start = str(i).index("data-coming-soon=")
+				end = str(i).index('data-event-code')
+				data = str(i)[start+18 : end-2]
+				
+				if data == "false":

```

Comment:
One doubt, when we are hitting the `now_showing` URL then why do we have a list of coming soon movies? Should we not filter only the ones that are currently running?
Or maybe display the 2 list sepratetly?

Reply:
It Displays Movies for which Advance Bookings are Opened. And Releasing on next Weekends.